packages/vue-cli-plugin-typescript/generator/index.js

```diff
@@ -12,6 +12,8 @@ module.exports = (api, options, rootOptions) => {
       '@coveo/search-token-server': '*',
       buefy: '^0.9.4',
       concurrently: '^5.3.0',
+      dotenv: '^9.0.2',
+      'fs-extra': '^10.0.0',
```

Comment:
It's because the generated project depends on these 2 packages.
Currently we were lucky because these packages are available though other packages, hence why it never failed.